README.md

```diff
@@ -128,8 +141,17 @@ All props accept arrays as values for mobile-first responsive styles.
 
 styled-system includes a higher order component to add style props to any component.
 
+**With `styled-components`**
+
+```jsx
+import hoc from 'styled-system/dist/styled-components/hoc'
```

Comment:
👍 fair warning I might change my mind later and rip out any library specific stuff